apps/tailwind-components/app/assets/css/theme/dark.css

```diff
@@ -31,24 +31,21 @@
   --background-color-button-outline-hover: var(--color-gray-900);
 
   --background-color-content: var(--color-gray-700);
-
```

Comment:
try to prevent these kind of changes because they distract the reviewer from what changes are essential